### PR TITLE
[TOPIC-BLE-LLCP] Bluetooth: controller: inclusive naming and added includes

### PR DESCRIPTION
**Pull request by kruithofa**
Some inclusive naming changes were already in the main branch, which resulted in compile errors
Some includes were missing, causing failures in CI
Some EDTT tests are failing, disabled pending investigation

Signed-off-by: Andries Kruithof <Andries.Kruithof@nordicsemi.no>

### CODE REVIEW

File: subsys/bluetooth/controller/ll_sw/ull_chan.c [hunk 1]
```diff
@@ -38,8 +39,11 @@
 static uint8_t map[5] = {0xFF, 0xFF, 0xFF, 0xFF, 0x1F};
 static uint8_t count = 37U;
 
+#if defined(CONFIG_BT_LL_SW_SPLIT_LLCP_LEGACY)
 static void chan_map_set(uint8_t const *const chan_map);
```
Comment (erbr-ot) [inline subsys/bluetooth/controller/ll_sw/ull_chan.c:43]: Why not remove the static version in the legacy build and simply use ull_chan_map_set?

**Review by erbr-ot — APPROVED** [review]
Only issue I have is re. chan_map_set()

**Comment by thoh-ot** [comment]
Wan't CI supposed to be green?

**Comment by kruithofa** [comment]
> Wasn't CI supposed to be green?

That was the expectation yes

**Comment by kruithofa** [comment]
So let's see if things are improved now

**Review by erbr-ot — APPROVED** [review]
Still the question re. chan_map_set() but that apart it looks just dandy to me

**Review by thoh-ot — APPROVED** [review]

**Pull request merged**